{{ cookiecutter.repo_name }}/requirements.txt

```diff
@@ -0,0 +1 @@
+spyder==3.1.3
```

Comment:
yes, I's the latest stable version, and I want to update dependencies using pyup #27 so dependencies should be pinned